quic/s2n-quic-crypto/src/cipher_suite/ring.rs

```diff
@@ -1,84 +1,130 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-macro_rules! key {
+macro_rules! fips_supported_key {
```

Comment:
can you put a comment describing how this is different from the regular key, it gets a little confusing understanding the difference between "fips_supported" and the `fips` feature flag